azure/eventprocessorhost/azure_blob_lease.py

```diff
@@ -48,6 +48,7 @@ def with_blob(self, blob):
         self.epoch = content["epoch"]
         self.offset = content["offset"]
         self.sequence_number = content["sequence_number"]
+        self.event_processor_context = content["event_processor_context"]
```

Comment:
This should probably be `content.get("event_processor_context") in case we're running against leases created by a previous version that don't have the extra data.

Reply:
You're right. Fixed.